atd-vze/.env

```diff
@@ -1,5 +1,5 @@
 REACT_APP_HASURA_ENDPOINT="https://vzd-staging.austinmobility.io/v1/graphql"
-#REACT_APP_HASURA_ENDPOINT="https://localhost:8080/v1/graphql"
+# REACT_APP_HASURA_ENDPOINT="http://localhost:8080/v1/graphql"
```

Comment:
i get a network error when i run locally and uncomment out this line and sometimes it takes me awhile to remember why, but its because this `https` needs to be changed to `http`. does this happen to anyone else? if not, i can change this back!

Reply:
it happens to me too, im good with this change

Reply:
All good with this change too! 🙏 I could even see us updating this so the staging endpoint is commented out so we default to local.